src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -109,7 +109,7 @@
     @FindBy(id = "btnLogout")
     private WebElement logoutButton;
 
-    @FindBy(xpath = "(//*[@class=\"htCore\"]/tbody/tr/td)[1]")
+    @FindBy(xpath = "(//*[@class=\"htCore\"]/tbody/tr/td[@class =\"enroll-handsontable\"])[1]")
     private WebElement spreadsheetFirstCell;
```

Comment:
there are 2 spreadsheets now. Which one are you referring to? And why only 1 but not the other?

Reply:
This WebElement is used to facilitate the test of filling in the students' details (can only be done by first selecting the first cell of the enroll spreadsheet).

Since the existing students' spreadsheet doesn't require interaction for tests (at least not in the current stage), I made the distinction between the two spreadsheets.

Reply:
Yep but you should still keep in mind that the other spreadsheet may need to be tested in the future and also to help reader understand what spreadsheet you are referring to by giving it a more specific name.